packages/cozy-harvest-lib/src/components/OAuthWindow.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+
+import { translate } from 'cozy-ui/react/I18n'
+
+import { prepareOAuth, checkOAuthData, terminateOAuth } from '../helpers/oauth'
+import withDomain from './hoc/withDomain'
+import Popup from './Popup'
+
+const OAUTH_POPUP_HEIGHT = 800
+const OAUTH_POPUP_WIDTH = 800
+
+/**
+ * OAuth window is responsible for managing the OAuth process on the client
+ * side, form storing info into local storage, load OAuth window and listen
```

Comment:
```suggestion
 * side, storing info into local storage, loading OAuth window and listening
```

I think a list would be clearer here.

- managing process...
- storing info...
- loading oauth window...
- listening for data..